README.md

```diff
@@ -14,8 +10,279 @@ But we need have a clear way to discover how you've been invited to a group with
+          We encrypt this announce message so as not to leak the groupKey AND to protect group membership.
+          <br />
+          For sympathetic replication we will therefor need a distinct type of announce message (TODO)
+        </div>
+      </details>
+
+
+## Flows
+
+### 1. Creating a group
+
+Staltz starts up his application.
+We assume he has already created his `group-invite` feed (following the spec above).
+In his application he creates a new "helsinki" group, which means he:
+1. creates a new symmetric `groupKey`
+2. create a content feed under a new shard (using the `groupKey` following the spec above)
+3. publishes an encrypted `group/init` message to that new "helsinki" feed
+4. publishes an encrypted `group/add-member` message to his "group-invite" feed
+      <details>
+        <summary>details</summary>
+        <div>
+          This helps new members quickly see he is a member of the group, and also ensures he has a copy of the groupKey persisted in his records (encrypted to him and the group)
+        </div>
+      </details>
+
+```mermaid
+graph TB
+
+root(root)
+v1(v1)
+4(4)
+d(d)
+group-invite(group-invite):::groupInvite
+helsinki(helsinki):::group
+
+subgraph Staltz
+  root --> v1 --> 4 --> helsinki
+           v1 --> d --> group-invite
+end
+
+classDef default stroke:none;
+classDef groupInvite fill: #BF2669, stroke:none, color:white;
+classDef group fill: #702A8C, stroke: none, color:white;
+```
+_Diagram showing Staltz feed state from his perspective_
+
+### 2. Group creator invites someone
+
+Staltz wants to invite his friend Arj to the group he set up, so he publishes a `group/add-member` message (which conatins the `groupKey`) to his "group-invite" feed.
+
+When Arj next stats up his application and replicates Staltz's feed (they are friends) he discovers the new `group/add-member` for him Staltz's "group-invite" feed (peers MUST replicate friends group-invite feeds).
+
+```mermaid
+graph TB
+
+rootA(root)
+v1A(v1)
+4A(4):::unreplicated
+dA(d)
+group-inviteA(group-invite):::groupInvite
+helsinkiA(helsinki):::unreplicated
+
+rootB(root)
+v1B(v1)
+9B(9)
+group-inviteB(group-invite):::groupInvite
+
+subgraph Staltz
+  rootA --> v1A --> 4A --> helsinkiA
+            v1A --> dA --> group-inviteA
+end
+
+subgraph Arj
+  rootB --> v1B --> 9B --> group-inviteB
+end
+
+classDef default stroke:none;
+classDef groupInvite fill: #BF2669, stroke:none, color:white;
+classDef group fill: #702A8C, stroke: none, color:white;
+classDef unreplicated opacity: 0.4, stroke: none;
+```
+_Diagram showing feed state of Arj and Staltz from Arj's perspective. The greyed out feeds show feeds that exist for Staltz but which Arj has yet to want to replicate_
+
+
+Assuming he accepts this invitation Arj then does the following:
+1. calculates the path to the "helsinki" group for staltz, and starts replicating it
+    - in this example `@statlz/v1/4/helsinki` (where "helsinki" is derived from the `groupKey`)
```

Comment:
```suggestion
```

Let's ignore paths for now unless you want to update the `ssb-meta-feeds-spec` repo with "paths". :)